tests/system/feature_store/test_feature_store.py

```diff
@@ -592,6 +592,36 @@ def test_nosql_no_path(self):
             1
         ].get_path().get_absolute_path() == fset.get_target_path("nosql")
 
+    @pytest.mark.parametrize("local", [True, False])
+    def test_ingest_with_format_run_project(self, local):
+        first_data = pd.read_csv(
+            os.path.relpath(str(self.assets_path / "testdata.csv"))
+        )
+        feature_set = fstore.FeatureSet(
+            name=f"fs-run_project-format-local-{local}",
+            entities=[Entity("patient_id")],
+            timestamp_key="timestamp",
+        )
+        artifact_path = mlrun.mlconf.artifact_path
+        targets = [
+            CSVTarget(name="labels", path=os.path.join(artifact_path, "file.csv"))
+        ]
+        feature_set.set_targets(targets=targets, with_defaults=False)
+        feature_set.plot(with_targets=True)
+        config_kwargs = {"run_config": fstore.RunConfig(local=True)} if local else {}
```

Comment:
Again - I may be missing something, but isn't this the other way around? If you don't provide a `RunConfig` then `ingest` will run locally. So, you need a `RunConfig` when `local=False`, not like you did here. What happens here is that you run locally, regardless of the `local` parameter.